<?xml version="1.0"?>

<robot name="autoarm">


  <link name="base_link">
    <visual>
      <geometry>
        <mesh filename="package://autoarm_description/meshes/joint.dae"/>
      </geometry>
    </visual>
  </link>


<link name="head_link">
    <visual>
      <geometry>
        <mesh filename="package://autoarm_description/meshes/head.dae"/>
      </geometry>
    </visual>
  </link>

<link name="arm_link">
    <visual>
      <geometry>
        <mesh filename="package://autoarm_description/meshes/arm.dae"/>
      </geometry>
    </visual>
  </link>

 <joint name="base_to_arm" type="revolute">
    <parent link="base_link"/>
    <child link="arm_link"/>
    <origin xyz="0 0 0" rpy="0 0 0"/>
    <axis xyz="0 1 0"/>
    <limit effort="1000.0" lower="-1.5" upper="1.5" velocity="0.5"/>
  </joint>

 <joint name="arm_to_head" type="revolute">
    <parent link="arm_link"/>
    <child link="head_link"/>
    <origin xyz="0 0 0.08" rpy="0 0 0"/>
    <axis xyz="0 1 0"/>
    <limit effort="1000.0" lower="-1.5" upper="1.5" velocity="0.5"/>
  </joint>







</robot>
